packet/models_cacher.go

```diff
@@ -41,6 +41,7 @@ type HardwareCacher struct {
 	ServicesVersion   ServicesVersion `json:"services"`
 	Instance          *Instance       `json:"instance"`
 	ProvisionerEngine string          `json:"provisioner_engine"`
+	Traceparent       string          `json:"traceparent"`
```

Comment:
is `traceparent` set it stone? It kind of bugs me that its not `trace_parent` in the json and `TraceParent` in the struct. Its not a big deal... just a minor annoyance really but if we can change it that would be great.

Reply:
Unfortunately, yes it is set in stone. It's hard-coded in the otel spec and libraries, and they make it _really_ hard to mess with, without adding a lot of ugly boilerplate. We're using `traceparent` and `Traceparent` all over the place and I don't think it should be camelcased.